package.json

```
@@ -17,6 +17,7 @@
     "dev": "nodemon --watch 'src/**/*.ts' --exec 'npm run build && npm run start'",
     "gcp-build": "npm run build",
     "generate:env": "terraform -chdir=infra apply -target=local_file.env_file",
+    "get:relayer:address": "NODE_ENV=development ts-node ./src/get-relayer-address.ts",
```

nit: `ts-node` is not in the dependencies so it's dependent on locally having ts-node in the path

```suggestion
    "get:relayer:address": "NODE_ENV=development npx ts-node ./src/get-relayer-address.ts",
```